daemon/license.go

```diff
@@ -9,26 +10,24 @@ import (
 
 //Checks for cloud license in nios
 func CheckForCloudLicense(objMgr *ibclient.ObjectManager) {
-	flag, err := CheckLicense(objMgr, "cloud")
+	err := CheckLicense(objMgr, "cloud")
 	if err != nil {
-		logrus.Fatal("error", err)
-	}
-	if !flag {
-		logrus.Fatal("Cloud License not available in Infoblox Appliance. Update and try again..")
+		logrus.Fatal("Check Cloud License : ", err)
```

Comment:
replace to "Error while checking for cloud license: "

Reply:
done